### PR TITLE
AAP-55586 - Fix ci - WIP

### PR DESCRIPTION
**Pull request by KB-perByte**
## Description

- What is being changed?
- Why is this change needed?
- How does this change address the issue?

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Test update
- [ ] Refactoring (no functional changes)
- [ ] Development environment change
- [ ] Configuration change

## Self-Review Checklist

- [ ] I have performed a self-review of my code
- [ ] I have added relevant comments to complex code sections
- [ ] I have updated documentation where needed
- [ ] I have considered the security impact of these changes
- [ ] I have considered performance implications
- [ ] I have thought about error handling and edge cases
- [ ] I have tested the changes in my local environment

### Steps to Test
1. 
2. 
3. 

### Required Actions

- [ ] Requires documentation updates
  
- [ ] Requires downstream repository changes
  
- [ ] Requires infrastructure/deployment changes
  
- [ ] Requires coordination with other teams
  
- [ ] Blocked by PR/MR: #XXX

### CODE REVIEW

File: tools/ansible/ansible.cfg [hunk 1]
```diff
@@ -1,2 +1,3 @@
 [defaults]
 collections_path = /tmp/collections
+allow_broken_conditionals = True
```
Comment (KB-perByte) [inline tools/ansible/ansible.cfg]: not needed, revert

File: Makefile [hunk 2]
```diff
@@ -74,7 +74,7 @@ collection-test: collection-install
 	echo 'gateway_password: $(GATEWAY_PASSWORD)' > /tmp/collections/ansible_collections/ansible/platform/tests/integration/integration_config.yml && \
 	cat /tmp/collections/ansible_collections/ansible/platform/tests/integration/integration_config.yml && \
 	cd /tmp/collections/ansible_collections/ansible/platform && \
-	  ansible-test integration --color yes --venv --requirements --coverage
+	  ansible-test integration --color yes --venv --requirements --coverage -vvvv
```
Comment (KB-perByte) [inline Makefile:77]: not needed, revert

File: tests/integration/targets/authenticator_maps_test/tasks/main.yml [hunk 3]
```diff
@@ -58,7 +58,8 @@
       ansible.builtin.assert:
         that:
           - fail is failed
-          - fail.msg == 'map_type is team but all of the following are missing: team, organization'
+          - "'team' in fail.msg"
```
Comment (KB-perByte) [inline tests/integration/targets/authenticator_maps_test/tasks/main.yml]: `{'triggers': ['Triggers must be a valid dict'], 'team': ['You must specify a team with the selected map type'], 'organization': ['You must specify an organization with the selected map type'], 'role': ['You must specify a role with the selected map type']}`

This is the error at present.

No way it asserted to 
`map_type is team but all of the following are missing: team, organization`
^ prior to September 25th

**Comment by rohitthakur2590** [comment]
Closing this one as  we are fixing CI issue here: https://github.com/ansible/ansible.platform/pull/86
